### PR TITLE
feat(CRO): Implement homepage CTA copy test #3

### PR DESCRIPTION
**Pull request by mwickett**
### What
Implements a CRO test for CTA text changes on the homepage hero

### Why
Context in this [Asana ticket](https://app.asana.com/0/0/1203017484612685/f)

### Screenshots
TK

CleanShot 2023-03-08 at 10 01 11@2x

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

**Comment by mwickett** [comment]
@alexcarpenter Here's a first pass at a basic client side implementation using context. It doesn't solve for flash while flags settle, yet - can you give this a peek and LMK if I'm roughly on the right track here? All feedback welcome 😃

**Comment by alexcarpenter** [comment]
@mwickett yeah, looks correct. Tested with a 50/50 split in dev env and it looks to be working as expected.

**Comment by alexcarpenter** [comment]
dev portal implementation https://github.com/hashicorp/dev-portal/pull/1718

File: components/subnav/index.jsx [hunk 1]
```diff
@@ -24,14 +24,17 @@ export default function DefaultSubnav({ menuItems }) {
           url: 'https://developer.hashicorp.com/terraform/downloads',
         },
         {
-          text:
-            flagBag.settled && flagBag.flags.tryForFree
-              ? 'Try for Free'
-              : 'Try Terraform Cloud',
+          text: renderVariant ? 'Try for Free' : 'Try Terraform Cloud',
```
Comment (dstaley) [inline components/subnav/index.jsx:27]: Isn't it `Try for free`?

**Review by dstaley — APPROVED** [review]

File: pages/home/index.tsx [hunk 2]
```diff
@@ -65,7 +65,7 @@ export default function Homepage({ data }): React.ReactElement {
         brand="terraform"
         heading={heroHeading}
         description={heroDescription}
-        ctas={heroCtas.map((cta) => {
+        ctas={heroCtas.map((cta, i) => {
```
Comment (dstaley) [inline pages/home/index.tsx:68]: Looks like this index isn't being used anymore.

**Review by dstaley — APPROVED** [review]

**Pull request merged**